src/pages/profiles/OperatorProfile.js

```diff
@@ -1,19 +1,52 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { toast } from 'react-toastify';
+import TableSkeleton from '../../components/SkeletonUIs/TableSkeleton.js';
+import { axiosBase } from '../../utils/Api.js';
 import Profile from './Profile.js';
 
 const OperatorProfile = () => {
+  const [userInfo, setUserInfo] = useState([]);
+  const [publicInfo, setPublicInfo] = useState([]);
+  const [loading, setloading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setloading(true);
+      try {
+        const response = await axiosBase.get('/accounts/profile');
+
+        setUserInfo(response.data.data.user);
+        setPublicInfo(response.data.data);
+        setloading(false);
+      } catch (error) {
+        toast(error.message, { type: 'error' });
+      } finally {
```

Comment:
add a toast for errors provided by the backend when the request has a fail status